apps/src/p5lab/poetry/PoetryLibrary.js

```diff
@@ -30,6 +31,9 @@ export default class PoetryLibrary extends CoreLibrary {
         font: 'Arial'
       },
       frameType: undefined,
+      text: {
+        highlightColor: null
+      },
```

Comment:
I added a `text` object. I was thinking we could add `isVisible` and `effects` to this object in a separate PR.

Reply:
It might be simpler to put this in the `font` property?

Reply:
yeah I thought about that too but I don't think it would make sense from a readability stand point.